src/engine/LastPlayedVersion.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using Godot;
+
+/// <summary>
+///   Loads and allows updating the last played Thrive version
+/// </summary>
+public static class LastPlayedVersion
```

Comment:
Why don't we add a settings value for last played version?

Reply:
What version was last played is unrelated to any game settings. And with https://github.com/Revolutionary-Games/Thrive/issues/4028 a value stored in settings wouldn't necessarily even correlate with the last played version. Kind of the same reason really as why the game startup failure detection uses a separate file as well.